test/utils/messaging.ts

```diff
@@ -1,183 +1,6 @@
-
-  console.log(
-      // eslint-disable-next-line
-    `Waiting for xchain messages to be relayed... L1 hash: ${l1Tx.transactionHash}, L2 tx hash: ${retryableTicket}, L2 auto redeem tx: ${redeemTransaction}`
-  );
-
-  const retryableTicketReceipt = await l2.waitForTransaction(
-      retryableTicket,
-      undefined,
-      1000 * 60 * 15,
-  );
-  console.log(retryableTicketReceipt);
-
-  //   expect(retryableTicketReceipt.status).to.equal(1);
-
-  const autoRedeemReceipt = await l2.waitForTransaction(
-      autoRedeem,
-      undefined,
-      1000 * 60,
-  );
-  console.log(autoRedeemReceipt);
-
-  //   expect(autoRedeemReceipt.status).to.equal(1);
-
-  const redemptionReceipt = await l2.getTransactionReceipt(redeemTransaction);
-  console.log(redemptionReceipt);
-
-  //   expect(redemptionReceipt.status).equals(1);
-  console.log('Xchain message arrived');
-}
-
-async function getInboxSeqNumFromContractTransaction(
-    l1Transaction: providers.TransactionReceipt,
-    inboxAddress: string,
-    provider: ethers.providers.BaseProvider,
-) {
-  const contract = new ethers.Contract(
-      inboxAddress,
-      IInbox__factory.createInterface(),
-      provider,
-  );
-  const iface = contract.interface;
-  const messageDelivered = iface.getEvent('InboxMessageDelivered');
-  const messageDeliveredFromOrigin = iface.getEvent(
-      'InboxMessageDeliveredFromOrigin',
-  );
-
-  const eventTopics = {
-    InboxMessageDelivered: iface.getEventTopic(messageDelivered),
-    InboxMessageDeliveredFromOrigin: iface.getEventTopic(
-        messageDeliveredFromOrigin,
-    ),
-  };
-
-  const logs = l1Transaction.logs.filter(
-      (log) =>
-        log.topics[0] === eventTopics.InboxMessageDelivered ||
-      log.topics[0] === eventTopics.InboxMessageDeliveredFromOrigin,
-  );
-
-  if (logs.length === 0) return undefined;
-  return logs.map((log) => BigNumber.from(log.topics[1]));
-}
-
-async function calculateL2TransactionHash(
-    inboxSequenceNumber: BigNumber,
-    provider: ethers.providers.BaseProvider,
-) {
-  const l2ChainId = BigNumber.from((await provider.getNetwork()).chainId);
-
-  return utils.keccak256(
-      utils.concat([
-        utils.zeroPad(l2ChainId.toHexString(), 32),
-        utils.zeroPad(bitFlipSeqNum(inboxSequenceNumber).toHexString(), 32),
-      ]),
-  );
-}
-
-function bitFlipSeqNum(seqNum: BigNumber) {
-  return seqNum.or(BigNumber.from(1).shl(255));
-}
-
-function calculateRetryableAutoRedeemTxnHash(requestID: string) {
-  return utils.keccak256(
-      utils.concat([
-        utils.zeroPad(requestID, 32),
-        utils.zeroPad(BigNumber.from(1).toHexString(), 32),
-      ]),
-  );
-}
-
-function calculateL2RetryableTransactionHash(requestID: string) {
-  return utils.keccak256(
-      utils.concat([
-        utils.zeroPad(requestID, 32),
-        utils.zeroPad(BigNumber.from(0).toHexString(), 32),
-      ]),
-  );
-}
 
 export function applyL1ToL2Alias(l1Address: string): string {
```

Comment:
Removed applyL1ToL2Alias def here and imported the function from `utils/arbitrum` in [b2c3601](https://github.com/livepeer/arbitrum-lpt-bridge/pull/81/commits/b2c360166f58790000ae93c1d0174a450deec258). Left `getL2SignerFromL1` since that is test focused so I think keeping it in the test directory makes sense (vs. the other helpers which can be used inside of and outside of tests).